SQL/database_changelog.txt

```diff
@@ -1,3 +1,23 @@
+28 June 2017, by oranges
+Added schema_revision to store the current db revision, why start at 3.0?
+
+because:
+15:09 <+MrStonedOne> 1.0 was erro, 2.0 was when i removed erro_, 3.0 was when jordie made all the strings that hold numbers numbers
+
+CREATE TABLE `schema_revision` (
+`major` TINYINT(3) UNSIGNED NOT NULL ,
```

Comment:
As in https://github.com/tgstation/tgstation/pull/28897#discussion_r124453561 I don't think we'll have to be concerned with it, but if need be then use `SMALLINT(5)` for 4.2 billion changes.

And we really really don't need to be worried then.